va/va.go

```diff
@@ -57,6 +58,17 @@ const (
 	// variables PEBBLE_VA_NOSLEEP resp. PEBBLE_VA_SLEEPTIME (see above).
 	defaultSleepTime = 5
 
+	// validateTimeoutEnvVar defines the environment variable name used to set the
+	// timeout for a validation attempt. Set this e.g. to 20 when you invoke Pebble if
+	// you wish the timeout for a validation attempt to be higher:
+	//   PEBBLE_VA_TIMEOUT=20 pebble
+	validateTimeoutEnvVar = "PEBBLE_VA_TIMEOUT"
```

Comment:
If you're fine with a simple increase in timeout, I can go ahead and do that. I just wasn't sure if that was an acceptable PR since it's literally only an issue when I'm running an entire end-to-end test suite on my laptop :)

Trying to think of any adverse effects of just increasing the timeout and not seeing any immediately.

Reply:
Increasing the default and removing the env var is a :+1: from me. 

I think in most contexts your approach is the better one. We can play a little fast and loose with Pebble to keep it simple because its strictly intended for test rigs where a few requests hanging around longer than you might like seems like a fair trade-off for less knobs.